adafruit_debouncer.py

```diff
@@ -96,38 +107,38 @@ def update(self, new_state=None):
                     self._state_changed_ticks = now_ticks
 
     @property
-    def interval(self):
+    def interval(self) -> float:
         """The debounce delay, in seconds"""
         return self._interval_ticks / _TICKS_PER_SEC
 
     @interval.setter
-    def interval(self, new_interval_s):
-        self._interval_ticks = new_interval_s * _TICKS_PER_SEC
+    def interval(self, new_interval_s: float) -> None:
+        self._interval_ticks = int(new_interval_s * _TICKS_PER_SEC)
```

Comment:
I think it's reasonable to add the use of `int()` but now I'm wondering if that would cause unintended issues because now the getter and setters don't necessarily match.  Have any thoughts, @FoamyGuy?  The interval would have to be unreasonably small to cause an issue, so using `int()` may mean the inclusion of a check, throwing `ValueError` if `new_interval_s` is too small?

Reply:
I'm not sure what other if any consequences could come from adding this int conversion. It does seem like it should only affect very small units of time below 1ms so maybe it's okay to round it off but I'm not sure. 

We would definitely want to test it to make sure no obvious issues arise.

Reply:
I don't see anything immediately problematic with it, but also don't quite understand the benefit of it.

to err on the side of caution I'm going to remove this `int()` change for now. We can add it back in a future PR if desired.